scripts/timer.r

```diff
@@ -0,0 +1,17 @@
+library(tictoc)
+
+# Time the original solution
+tic("main")
+source("main.R")
+toc()
+
+# Time the parallel computing solution
+tic("parallel_computing")
+source("scripts/parallel_computing.R")
```

Comment:
Make sure that you have the correct path in the working directory. Both the file main and parallell_computing.r are in the same folder. Therefore the path should be "scripts/YOUR_PATH" for all files.